### ISSUE TITLE
TabbedInterface nested with ChatInterface, the height of the ChatInterface is always 200px

### ISSUE THREAD

**Issue by fanjianing** [gradio-app/gradio#7989]
### Describe the bug

When trying the TabbedInterface nested with ChatInterface, the height of the ChatInterface is always 200px. How to solve it?

Code Demo 1:
```
with gr.Blocks(fill_height=True) as block:
    gr.TabbedInterface(
    [moonshot_chat, claude3_chat],
    ["Moonshot", "Claude3 Sonnet Chat"],
    title="LLM Console",
    )
block.launch()
```

Code Demo 2:
```
app = gr.TabbedInterface(
    [moonshot_chat, claude3_chat],
    ["Moonshot", "Claude3 Sonnet Chat"],
    title="LLM Console",
    )
app.launch()
```

No matter what kind of code it is, it will always be this fixed height. How can it be made the same as the browser height?

![image](https://github.com/gradio-app/gradio/assets/5792689/0ace75c8-b0b7-43fb-9b01-bbcc35e7d74a)

I tried a method of replacing css, it worked! but the id of the corresponding element in each tab is different, which is very inconvenient. 

```
app = gr.TabbedInterface(
    [moonshot_chat, claude3_chat],
    ["Moonshot", "Claude3 Sonnet Chat"],
    title="LLM Console",
    css=".block  .svelte-12cmxck { height:calc(100vh - 380px)!important; }"
    )
```

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr

```

### System Info

```shell
gradio version = 4.26.0
```

### Severity

I can work around it

**Comment by xkbang** [comment]
Same here

**Comment by NatromeTex** [comment]
this work around works with tabbedInterface but not with normal tabs.

**Comment by Rijgersberg** [comment]
Same problem!

Here is a stand-alone minimal code sample:
```python
import gradio as gr


def process(history, message):
    return "Hello world"

chat_interface = gr.ChatInterface(
    fn=process,
)

with gr.Blocks() as second_interface:
    textbox = gr.Textbox()

demo = gr.TabbedInterface(
    interface_list=[chat_interface, second_interface],
)

demo.launch()
```

**Comment by abidlabs** [comment]
Closed via #9571
